src/fragments/lib/auth/common/mfa/flows.mdx

```diff
@@ -101,9 +113,49 @@ import iosConfirmSignInTOTP from "/src/fragments/lib/auth/ios/sms/confirm_sign_i
 
 <Fragments fragments={{ios: iosConfirmSignInTOTP}} />
 
+import android9 from "/src/fragments/lib/auth/android/sms/confirm_sign_in.mdx";
+
+<Fragments fragments={{android: android9}} />
+
 After a user has been signed in, call `updateMFAPreference` to record the MFA type as enabled for the user
 and optionally set it as preferred so that subsequent logins default to using this MFA type.
 
+<InlineFilter filters={['android']}>
+<BlockSwitcher>
+<Block name="Java">
+
+```java
+if (Amplify.Auth.getPlugin("awsCognitoAuthPlugin") instanceof AWSCognitoAuthPlugin) {
+            AWSCognitoAuthPlugin plugin = (AWSCognitoAuthPlugin) Amplify.Auth.getPlugin("awsCognitoAuthPlugin");
+            plugin.updateMFAPreference(
+                    MFAPreference.ENABLED,
+                    null,
+                    () -> Log.i("AuthQuickstart", "MFA preference updated successfully"),
+                    e -> Log.e("AuthQuickstart", "Failed to update MFA preference.", e)
+            );
+        }
+```
+
+</Block>
+<Block name="Kotlin - Callbacks">
+
+```kotlin
+if (Amplify.Auth.getPlugin("awsCognitoAuthPlugin") is AWSCognitoAuthPlugin) {
+    val plugin = Amplify.Auth.getPlugin("awsCognitoAuthPlugin") as AWSCognitoAuthPlugin
```

Comment:
Here we can do an `as?` instead of an explicit check.

Reply:
e.g. like here: https://github.com/aws-amplify/docs/pull/5828/files#diff-47b6d217f1b280dd051d6b0974e691d716b76a41649f7a04d8854240df68fb10R18-R22

Reply:
done